rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/180_percentiles_tdigest_metric.yml

```diff
@@ -62,27 +62,47 @@ setup:
               percentiles:
                 field: double_field
 
-
-  - match: { hits.total: 4 }
+  - match:  { hits.total: 4 }
   - length: { hits.hits: 4 }
-  - match:
-      aggregations.percentiles_int.values:
-        "1.0": 2.5
-        "5.0": 8.500000000000002
-        "25.0": 38.5
-        "50.0": 76.0
-        "75.0": 113.5
-        "95.0": 143.49999999999997
-        "99.0": 149.5
-  - match:
-      aggregations.percentiles_double.values:
-        "1.0": 2.5
-        "5.0": 8.500000000000002
-        "25.0": 38.5
-        "50.0": 76.0
-        "75.0": 113.5
-        "95.0": 143.49999999999997
-        "99.0": 149.5
+
+  # - match:
```

Comment:
The fixes made by @fxdgear in #26905, and my subsequent fixes in #27355 worked nicely for me in Ruby. I must admit I like more the original notation. Also, some tests in the proposed notation (`aggregations.percentiles_int.values.5\.0`) fail for me, not sure how the Ruby runner handles it, must debug...

Reply:
The test runners are supposed to support dots escaped with `\`.

The original notation is a non-starter for me, for instance Perl gets back `143.5` and keeps it as that value, rather than converting to `143.49999999999997`.  But then it sees `1.0` in the response and reads that as `1`(then the test fails when it tries to compare it to `1.0`).

I can do numeric comparisons when there is a single numeric-like value on the right, but not with an object.

Reply:
What I could do (keeping the key-to-numeric-value format) is allow for approximate numeric matching (ie within a certain tolerance) so that I can remove the `lte`/`gte`

Reply:
I'm thinking we may just want to remove the values like `143.49999999999997`.  I didn't think about it at the time when I added the test, but these are likely subject to how each language interprets json floats and internal float vs double, etc.

Would simplify everyone's life if we stuck to nice "discrete" floats (`149.5`, etc).  The point of these REST tests are to just do rough validation of the body, not really the values, so I think it's fine if we omit the problematic percentiles.